### ISSUE TITLE
enhancement request - Ostinato should allow users to create LACP bundles

### ISSUE THREAD

**Issue by anubisg1** [pstavirs/ostinato#324]
Something that i use all the times, are LACP bundles.

Example of setup:

2 physical servers with 4x 10Gbps nic cards running one drone each
1 VM running the ostinato GUI, and connecting to the drones are remote port-groups .

i would like to never login or play with the servers where the drones are running after the intial setup.
i would like to only move cables around to match the topology being tested.

sometimes LACP is required for testing, now i need to setup within the OS where the drone is running, then the UI would pick up the bundle interface.

the idea is that once a drone is attached to the UI, i could manipulate any property of the interfaces made available to the drone.

i would like for example, to setup LACP bundles from within the UI as you can do with other traffic generators like spirent.
see link below

[https://support.spirent.com/SpirentCSC/SC_KnowledgeView?id=SOL12381#:~:text=Spirent%20TestCenter%3A%20How%20do%20I%20setup%20LACP%20traffic%20in%20Spirent%20TestCenter%3F,-Environment%2FVersions&text=Select%20the%20ports%20that%20need,by%20configuring%20the%20necessary%20parameters.](url)

i understand that this might not be trivial, as different OS handles networking differently, not just between windows/linux but also between linux distributions. nonetheless i think this would be a major improvement for which i know many people willing to pay for

**Comment by pstavirs** [comment]
Yes, this is a non-trivial feature requiring per-platform code and also refreshing the port list dynamically (currently the port list is fetched only at application startup). For those reasons, I'm not inclined to work on it unless there's a lot of interest for it or someone is willing to sponsor upfront development of the feature.

For now keeping the request open to gauge interest for this request.

**Comment by anubisg1** [comment]
not sure if this is the right way, but.. what would be the investment required if i wanted to sponsor?

**Comment by pstavirs** [comment]
@anubisg1 Please send me an email regarding sponsorship and we can discuss there.
